front/src/store/modules/ReristrationUser.store.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+
+export default {
+  actions: {
+    AddUser ({ commit }, newUser) {
+      axios({
+        method: 'post',
+        url: 'https://localhost:44333/api/user',
+        data: {
+          userName: newUser.userName,
+          email: newUser.email,
+          userPassword: newUser.userPassword
+        }
+      }).then(response => {
+        if (Number(response.data)) {
+          commit('updateMailboxBusy', !response.data);
+          commit('updateVisibleRegistrationForm', !response.data);
+          commit('updateVisibleRegistrationSuccess', response.data);
```

Comment:
you see, it has problems. 
your store logic is connected with details of component. each change in design or naming in component or other things will cause the changes in store. it is high cohesion 
it is better to store just something like isRegistrationSuccess and based on this to make some other things in your components. Remember, store is more business logic layer. variable name should reflect its meaning not form

Reply:
I do not understand, please read more. Example?